themes/bootstrap3/js/resultcount.js

```diff
@@ -0,0 +1,24 @@
+/* global VuFind */
+
+VuFind.register('resultcount', function resultCount() {
+  function init() {
+    $('ul.nav-tabs li a').each(function queryResultCount(){
+      var $this = $(this);
+      if ($this.attr('href') !== undefined) {
+        var queryString = $this.attr('href').replace(/^.*\?/, '');
+        var source = $this.data('source');
+        jQuery.ajax({
```

Comment:
Any particular reason for using `jQuery` instead of `$` here? I think we usually use `$`, so it might be more stylistically consistent.

Reply:
No, we can use $ here.